### ISSUE TITLE
Mac OS standard shortcut

### ISSUE THREAD

**Issue by jerome-godbout-lychens** [sourcegit-scm/sourcegit#1716]
Hi, nice app by the way. I wish the hide application would react to the cmd+h keystroke like all other Mac OS application. This break my keyboard nav into my day to day workflow, I cannot count the number of time I hit that shortcut and have a 2 min and thinking my keyboard is dead. Not a big thing, but would make this app more seamless under Mac OS.

**Comment by yyfearth** [comment]
Also cmd+m
Zoom cmd+ cmd-
Enter fullscreen cmd+ctrl+f
And cmd+r should behave same as F5 for refresh.

Beside that the app crashed few times when I try snap window and click title bar few time.

**Comment by love-linger** [comment]
> Beside that the app crashed few times when I try snap window and click title bar few time.

The macOS package has some known issues which has been fixed in current develop branch (will release on next Monday)

And you can find the crash log in `App Data Storage Directory` (you can open that folder from the app's main menu `Open Data Storage Directory`)

---

EDIT:

I've reproduced this issue, and confirmed it's a BUG belongs to `AvaloniaUI`. I see there's already a fix to it. https://github.com/AvaloniaUI/Avalonia/pull/19445

**Comment by love-linger** [comment]
And I've confirmed that all drag and drop events on macOS has this issue

**Comment by love-linger** [comment]
The `⌘+H` hotkey has been added.

**Comment by love-linger** [comment]
> Beside that the app crashed few times when I try snap window and click title bar few time.

Fixed in `2025.31`
